src/bin/plop.js

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env node
 const args = process.argv.slice(2);
-const {Plop, run} = require('../dist/plop');
+const {Plop, run} = require('../plop');
```

Comment:
The bin has now been moved to the `src/bin` folder, and uses the `src` `require` so webpack can link them up